frontend/src/components/Details/Details.jsx

```diff
@@ -0,0 +1,51 @@
+import { useDispatch, useSelector } from 'react-redux'
+import './Details.css'
+import Loading from './Loading'
+const Details = () => {
+    // const dispatch = useDispatch()
+    // const data = useSelector(state => state.details)
+    // const details = data.details
+    // const reservation_message = data.reservation_message
```

Comment:
[REQUIRED] In order to get a clean code, kindly erase all the unused code because it doesn't look good in the project files.

Reply:
Those code will still be used. I commented them because I did not have a store setup. Once there is a redux store, I'll uncomment them.